### PR TITLE
User property operations

### PR DESCRIPTION
**Pull request by djih**

### CODE REVIEW

File: Amplitude/AMPDatabaseHelper.m [hunk 1]
```diff
@@ -97,6 +98,9 @@ - (BOOL)createTables
         NSString *createEventsTable = [NSString stringWithFormat:CREATE_EVENT_TABLE, EVENT_TABLE_NAME, ID_FIELD, EVENT_FIELD];
         success &= [db executeUpdate:createEventsTable];
 
+        NSString *createIdentifysTable = [NSString stringWithFormat:CREATE_EVENT_TABLE, IDENTIFY_TABLE_NAME, ID_FIELD, EVENT_FIELD];
```
Comment (curtisliu) [inline Amplitude/AMPDatabaseHelper.m]: seems a little risky to reuse CREATE_EVENT_TABLE here

File: Amplitude/Amplitude.m [hunk 2]
```diff
@@ -991,32 +1082,37 @@ - (void)startSession
...
+    if (identify == nil || [identify.userPropertyOperations count] == 0) {
+        return;
+    }
+    [self logEvent:IDENTIFY_EVENT withEventProperties:nil withApiProperties:nil withUserProperties:identify.userPropertyOperations withTimestamp:nil outOfSession:NO];
 }
 
-- (void)setUserProperties:(NSDictionary*) userProperties replace:(BOOL) replace
+#pragma mark - configurations
+
+- (void)setUserProperties:(NSDictionary*) userProperties
```
Comment (curtisliu) [inline Amplitude/Amplitude.m]: it breaks backwards compatibility to remove the replace

**Comment by curtisliu** [comment]
A lot of code, but LGTM.

File: Amplitude/Amplitude.m (hunk 2 above)
Comment (djih) [inline Amplitude/Amplitude.m]: I didn't remove the one with the replace. I just moved it to line 1113, and made it call this one.

Comment (curtisliu) [inline Amplitude/Amplitude.m]: Ah ok :+1:

**Pull request merged**